### ISSUE TITLE
Feature: SublimeText2 Plugin

### ISSUE THREAD

**Issue by jimthedev** [jimbojsb/emogrify#1]
The only thing that could make Emogrify more awesome is if there was a ST2 plugin for it. Probably will be a fork / separate project, but until then, figure I'd throw it out there to see what others think.

**Comment by jimbojsb** [comment]
Yeah it'd have to be ported to python

**Comment by jimthedev** [comment]
Yes. Or it would require PHP5 as a prerequisite like this does: https://github.com/welovewordpress/SublimeHtmlTidy

I have a command line php script that can parse simple arguments and returns the resultant html to stdout. All the python plugin would need to do is start a new php process and put the result from stdout to a new sublime tab.

**Comment by jimthedev** [comment]
[Here is the gist](https://gist.github.com/5534306) that shows how it could be done. Obviously you need to have emogrifier.php in the same directory and need to point it at your html and css files (Currently only works in the same directory, needs modification).

Command to output to stdout is: php emo.php index.html style.css
Command to output to file is: php emo.php index.html style.css index-inline.html

**Comment by jimthedev** [comment]
https://gist.github.com/jcummins/5534547

The gist of my ~/.emo/aliases file. If you load this file in your .profile or bashrc then you can use the global command: emo index.html style.css

**Comment by jimthedev** [comment]
Additionally, you can do:
brew install browser

Then do:
emo index.html style.css | browser

This will pipe the html directly into the browser for viewing.
